doc_source/cni-upgrades.md

```diff
@@ -63,4 +65,75 @@ Any changes you've made to the plugin's default settings on your cluster can be
 
       ```
       kubectl apply -f aws-k8s-cni.yaml
-      ```
\ No newline at end of file
+      ```
+**Custom upgrade scenarios when CNI custom networking in place**
+
+The upgrade might be an involved process when custom networking is in place. While there are efforts underway to make this process seamless, the best practice is to consider backup, analyze, and update resources to minimize undesirable application downtime\. It is important to understand the exact steps might vary across different versions\. Below steps work best when only an image change is required.
+
+***Backup***
+  + Take a backup of current CNI daemonset\.
+
+    ```
+    kubectl get daemonset aws-node -n kube-system -o yaml > aws-k8s-cni-old.yaml
+    ```
+  + Download the manifest file\.
```

Comment:
Maybe here we can also link the release page - https://github.com/aws/amazon-vpc-cni-k8s/releases